### PR TITLE
vertically merged cells

### PR DESCRIPTION
**Pull request by chessmith**
I fixed a bug preventing worksheets with vertically merged cells from loading properly and added a new test. The regular expression used to find cell elements did not properly recognize the self-closing tags used for vertically merged cells.

### CODE REVIEW

File: test/test_readxl.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def test_AllSheetsRead(self):
         db_ws_names = DB.ws_names
         db_ws_names.sort()
         # test 10+ sheets to ensure sorting matches correctly
-        true_ws_names = ['empty', 'types', 'scatter', 'length', 'sheet_not_to_read',
+        true_ws_names = ['empty', 'types', 'scatter', 'merged_cells', 'length', 'sheet_not_to_read',
```
Comment (chessmith) [inline test/test_readxl.py]: Added a new worksheet to testbook.xlsx

**Comment by PydPiper** [comment]
oh excellent edge case, thank you for writing up a test for it as well
